### PR TITLE
string-manipulation - Lamberto Martinez

### PR DESCRIPTION
**Pull request by Lambpato**
Screenshot 2023-02-03 at 3 12 19 PM

### CODE REVIEW

File: string-manipulation-1/src/get-first-char.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/* exported getFirstChar */
+
+function getFirstChar(string) {
+  for (var i = 0; i < string.length; i++) {
+    if (i === 0) {
+      return string[i];
+    }
+  }
+}
```
Comment (shawnkost) [inline string-manipulation-1/src/get-first-char.js:9]: You don't need a loop here. You can just return `string[0]`

File: string-manipulation-1/src/get-words.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/* exported getWords */
+
+function getWords(string) {
+  return string.split(' ').filter(Element => Element);
+}
```
Comment (shawnkost) [inline string-manipulation-1/src/get-words.js:5]: Can you explain to me why you used the `filter` method here, and what it does?

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Lamberto! I left a comment regarding your usage of the `filter` method. If you could please respond to my comment, and ping me when you respond so I can take a look please!

File: string-manipulation-1/src/get-first-char.js (hunk 1 above)
Comment (Lambpato) [reply string-manipulation-1/src/get-first-char.js:9]: 🫡

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**